cli/alias.txt

```diff
@@ -0,0 +1,5 @@
+alias rscli='java -cp "/Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar" io.apicurio.registry.cli.EntryCommand'
+java -cp "/Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar" picocli.AutoComplete -n rscli io.apicurio.registry.cli.EntryCommand
+https://github.com/remkop/picocli/blob/master/picocli-shell-jline2/README.md
+java -jar target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar
+alias rsmain='java -jar /Users/alesj/projects/redhat/apicurio-registry/cli/target/apicurio-registry-cli-2.0.0-SNAPSHOT.jar'
```

Comment:
What is this file?  Since it has `/Users/alesj` paths in it I'm assuming it shouldn't be included in the PR?

Reply:
Yeah, this should go into README -- it's the hints you need to easily run the CLI.